cli/cmd/inject_test.go

```diff
@@ -46,3 +46,49 @@ func TestInjectYAML(t *testing.T) {
 		})
 	}
 }
+
+func TestRunInjectCmd(t *testing.T) {
+	testCases := []struct {
+		inputFileName        string
+		stdErrGoldenFileName string
+		stdOutGoldenFileName string
+		exitCode             int
+	}{
+		{
+			inputFileName:        "inject_gettest_deployment.bad.input.yml",
+			stdErrGoldenFileName: "inject_gettest_deployment.bad.golden",
+			exitCode:             1,
+		},
+		{
+			inputFileName:        "inject_gettest_deployment.good.input.yml",
+			stdOutGoldenFileName: "inject_gettest_deployment.good.golden.yml",
+			exitCode:             0,
+		},
+	}
+
+	for i, tc := range testCases {
+		t.Run(fmt.Sprintf("%d: %s", i, tc.inputFileName), func(t *testing.T) {
+			errBuffer := &bytes.Buffer{}
+			outBuffer := &bytes.Buffer{}
+
+			in, err := os.Open(fmt.Sprintf("testData/%s", tc.inputFileName))
```

Comment:
`testdata`

Reply:
good catch. It's weird that it still proceed through the test even though the directory name was different than intended.